scripts/install-linux-deps.sh

```diff
@@ -19,6 +19,7 @@ env DEBIAN_FRONTEND=noninteractive sudo apt install --yes --quiet \
     git \
     jq \
     lcov \
+    libboost-all-dev \
```

Comment:
I thought I saw that debian also packages each header-only boost as dev, but seems I was mistaken. very very annoying..